scripts/docker.sh

```diff
@@ -55,6 +55,15 @@ deploy_branch() {
   docker push "coralproject/talk:${CIRCLE_BRANCH}"
 }
 
+deploy_commit() {
+  SHORT_GIT_HASH=$(echo $CIRCLE_SHA1 | cut -c -6)
+  echo "==> tagging commit ${SHORT_GIT_HASH}"
+  docker tag "coralproject/talk:latest" "${GCR_IMAGE_NAME}:${SHORT_GIT_HASH}"
```

Comment:
What do you think about setting the  `CIRCLE_TAG` when we call `docker.sh` so that we don't rely on the defaults of `coralproject/talk:latest`?

I'm sure this works! It could be worthwhile to be specific in the tagging when we build the image instead of relying on an indirect re-tag here though.

Let me know your thoughts 😄

Reply:
I think that's a good idea, but because this is modeled after `deploy_latest()` and `deploy_branch()`, both of which include that string hard-coded already, I think it's a good idea to push this as is and abstract all of them together as a separate improvement if it becomes necessary.

Reply:
Thee `coralproject/talk:latest` was chosen intentially as it's the tag that's created when the `docker build ...` happens elsewhere in this script. This script creates many different tags, and it's not in this case linked to the branch name or tag name.